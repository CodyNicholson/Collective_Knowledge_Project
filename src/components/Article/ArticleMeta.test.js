//This file was begun using code from Link.react.test.js, from the Jest Testing React Apps Guide
//https://facebook.github.io/jest/docs/en/tutorial-react.html
import React from 'react';
import ArticleMeta from './ArticleMeta';
import renderer from 'react-test-renderer';
import { MemoryRouter } from 'react-router-dom';
import { ArticleActions } from './ArticleActions';

var authorData = {username:"mikey", image:null};
var articleData = {author: authorData, createdAt: new Date(1997, 6, 24, 14, 39, 7)}

test('Tags module test', () => {
  const component = renderer.create(
    <MemoryRouter>
	    <ArticleActions>
	    	<ArticleMeta
	              article={articleData}
	              canModify={false}> </ArticleMeta>
	    </ArticleActions>
    </MemoryRouter>
  );

  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});