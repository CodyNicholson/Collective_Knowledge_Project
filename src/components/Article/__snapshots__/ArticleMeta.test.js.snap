// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Tags module test 1`] = `<span />`;
